bulbs/super_features/viewsets.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import views, viewsets, status
+from rest_framework.response import Response
+from rest_framework.permissions import IsAdminUser
+
+from bulbs.api.permissions import CanEditContent
+from bulbs.super_features.models import ContentRelation
+from bulbs.super_features.serializers import ContentRelationSerializer
+from bulbs.super_features.utils import get_superfeature_model, get_superfeature_partial_serializer
+
+
+SUPERFEATURE_MODEL = get_superfeature_model()
+SUPERFEATURE_PARTIAL_SERIALIZER = get_superfeature_partial_serializer()
+
+
+class ContentRelationViewSet(viewsets.ModelViewSet):
+
+    model = ContentRelation
+    serializer_class = ContentRelationSerializer
+    permission_classes = (IsAdminUser, CanEditContent,)
+
+
+class RelationViewSet(views.APIView):
+
+    permission_classes = (IsAdminUser, CanEditContent,)
+
+    def get(self, request, pk):
+        child_ids = ContentRelation.objects.filter(
+            parent_id=pk
+        ).values_list(
+            'child_id', flat=True
+        ).order_by('ordering')
+        children = SUPERFEATURE_MODEL.objects.filter(id__in=child_ids)
+        result = SUPERFEATURE_PARTIAL_SERIALIZER(children, many=True)
```

Comment:
You only need to check `is_valid` if you're using the `data=` kwarg when instantiating the serializer. In this case, this will just return an empty list if there are no children.